OpenSim/Actuators/Millard2012EquilibriumMuscle.cpp

```diff
@@ -878,9 +838,10 @@ calcMuscleDynamicsInfo(const SimTK::State& s, MuscleDynamicsInfo& mdi) const
         // Compute dynamic quantities.
         double a = SimTK::NaN;
         if(!get_ignore_activation_dynamics()) {
-            a = clampActivation(getStateVariableValue(s, STATE_ACTIVATION_NAME));
+            a = getActivationModel().clampActivation(
+                    getStateVariableValue(s, STATE_ACTIVATION_NAME));
```

Comment:
now with a separate activationModel it would make sense for the activation state to be part of it and also take its activation time constant properties with it.